backend/src/modules/votes/services/create.vote.service.ts

```diff
@@ -15,8 +16,22 @@ export default class CreateVoteServiceImpl implements CreateVoteService {
 		private boardUserModel: Model<BoardUserDocument>
 	) {}
 
-	incrementVoteUser(boardId: string, userId: string) {
-		return this.boardUserModel
+	private async voteEnable(boardId: string, userId: string): Promise<boolean> {
+		const board = await this.boardModel.findById(boardId).exec();
+		if (!board) {
+			throw new NotFoundException('Board not found!');
+		}
+		const maxVotes = board?.maxVotes as Number;
+
+		// myUser
+		const myUser = await this.boardUserModel.find({ board: boardId, user: userId });
```

Comment:
Try to avoid personal names when naming variables

```suggestion
		const userFound = await this.boardUserModel.find({ board: boardId, user: userId });
```
or

```suggestion
		const boardUser = await this.boardUserModel.find({ board: boardId, user: userId });
```

Reply:
thanks